readme.md

```diff
@@ -73,6 +73,14 @@ A sample configuration for the Service plugin:
             maximumVersion = "1.5.x"
             recommendedVersion = "1.3.0"
         }
+        productDependency {
+            productGroup = "other-group2"
+            productName = "other-service2"
+            // Automatically detect the constraints based on the runtime configuration
+            // API jars that publish recommended product dependencies.
+            // See the recommended product dependencies plugin section below.
+            detectConstraints = true
```

Comment:
how does this detection happen?  Does this read all jars in the runtime config, pull out any product recommendations they have, and use that as this recommendation?

What happens if there are conflicts in the recs in the runtime jars?